### ISSUE TITLE
Feature Request: save as new

### ISSUE THREAD

**Issue by mesqueeb** [adamreisnz/replace-in-file#120]
I'd love to be able to set a `toFiles` option.

Eg.

```js
const options = {
  files: 'path/to/files/*.html',
  toFiles: fileName => `new/path/${fileName}`
}
```

Cheers!!

**Comment by akaguny** [comment]
Look good =)
I think that we need to look at gulp/grunt tasks for cases and api because it is "standard"

**Comment by dbhat6** [comment]
@akaguny Would love this feature to save the new file content into a few file!

**Comment by oliverjessner** [comment]
Is there still no option?

**Comment by adamreisnz** [comment]
No option to save to a new file yet, if anyone wants to open a PR I'll be happy to include it.

**Comment by adamreisnz** [comment]
This has been implemented in 8.0.0 which will be released shortly
